src/frontend/src/lib/components/hero/Hero.svelte

```diff
@@ -37,6 +38,8 @@
 
 				<Balance />
 			</div>
+		{:else}
+			<TotalBalance/>		
```

Comment:
what's the logic here? why summary displays a summary else "totalbalance"?
an animation is also missing.